front/src/modules/timesStops/TimesStopsTable.tsx

```diff
@@ -0,0 +1,127 @@
+import { useMemo } from 'react';
+
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from '@tanstack/react-table';
+import { useTranslation } from 'react-i18next';
+
+import { Loader } from 'common/Loaders/Loader';
+import { formatLocalTime } from 'utils/date';
+
+import type { TimesStopsRowNew } from './types';
+
+type TimesStopsTableProps = {
+  rows: TimesStopsRowNew[];
+  dataIsLoading: boolean;
+};
+
+const columnHelper = createColumnHelper<TimesStopsRowNew>();
+
+const TimesStopsTable = ({ rows, dataIsLoading }: TimesStopsTableProps) => {
+  const { t } = useTranslation('translation', { keyPrefix: 'timeStopTable' });
+
+  const columns = useMemo(
+    () => [
+      columnHelper.display({
+        id: 'index',
+        header: '#',
+        cell: (info) => info.row.index + 1,
```

Comment:
It would be better to store the index of the row in the type `TimesStopsRowNew`, this way if we hide some columns (for instance all the non-requested points on path), we will still have the right index
```suggestion
        cell: (info) => info.row.original.index + 1,
```